cli/src/main.rs

```diff
@@ -143,10 +146,7 @@ async fn main() -> Result<()> {
 			util::telemetry::GAME_ID.get(),
 			"cli_error",
 			Some(|event: &mut async_posthog::Event| {
-				event.insert_prop(
-					"errors",
-					err.chain().map(|e| e.to_string()).collect::<Vec<_>>(),
-				)?;
+				event.insert_prop("errors", format!("{}", err))?;
 				Ok(())
 			}),
 		)
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines 143 to 154]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->@MasterPtato it would be good if you could confirm the behaviour here. It seems that anyhow let you chain multiple errors together then iterate over them to get all of their readable errors into a vec. Does this look like an ok replacement for it?<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/rivet-gg/cli/111?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
lgtm